pkg/cluster/nodes/util.go

```diff
@@ -31,14 +31,16 @@ func GetControlPlaneEndpoint(allNodes []Node) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	// if there is no external load balancer
+	// if there is no external load balancer return the empty string
 	if node == nil {
 		return "", nil
 	}
-	// gets the IP of the load balancer
+
+	// get the IP and port for the load balancer
```

Comment:
TODO: seems to be issues if we use the mapped port instead of 6443. need to look into that, this is fine for now though (we just "get" the fixed port)